src/lwc/geFormField/geFormField.js

```diff
@@ -523,41 +506,16 @@ export default class GeFormField extends LightningElement {
         }
 
         // reset lookups and recordtype fields
-        if (this.isLookupRecordType || this.isLookup) {
+        if (this.isLookup) {
```

Comment:
@psadie-sfdo is this function needed at all anymore?

Reply:
We should check in early next week, but my thoughts:

* the first block `if (setDefaults) {` should be updated to `if (!this.isPicklist && setDefaults) {` since picklists value will now be reset in the renderer.
* the second block `if (this.isLookup) {` should be removed because I don't think that `querySelector('[data-id="inputComponent"]')` is needed anymore 
* the third block `if (this.isPicklist) {` can also be removed since picklists get their reset value from formState now